### ISSUE TITLE
missing headers from tk/generic tk/unix

### ISSUE THREAD

**Issue by looooo** [conda-forge/tk-feedstock#4]
for netgen it's necesarry to include the headers from tk/generic and tk/unix. Would it be possible to include them? 

Something like this at the end of the build.sh would help a lot:
```
cp $SRC_DIR/tk${VER}/generic/*.h ${PREFIX}/include/
cp $SRC_DIR/tk${VER}/unix/*.h ${PREFIX}/include/
```

**Comment by jjhelmus** [comment]
@looooo This headers can most definitely be included, although it might take me a bit to get to this. If you want to submit a PR with this change I'd be happy to review and would speed up the process.

**Comment by looooo** [comment]
thanks, I will try to do this.

**Comment by jjhelmus** [comment]
closed in #5

**Comment by looooo** [comment]
thanks for fixing this
